### PR TITLE
Component added

### PR DESCRIPTION
**Pull request by dotemacs**
As discussed with @danielcompton added component based, REPL friendly flow to the app.

Added JUXT's `aero` for the configuration instead of environ.
Updated developer instructions.
Added reloaded workflow.

Also removed duplicate require of **resp.util.response**:
https://github.com/deps-app/versions/commit/f4a7ce3d6dc1fe4bd9ff44ff7af7bb9000653daf#diff-bed5708349f92d9b4bbd74f22e526340R23 as it was already present: https://github.com/deps-app/versions/blob/master/src/jarkeeper/core.clj#L10

### CODE REVIEW

File: resources/config.edn [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{:redis-uri #profile {:dev "redis://localhost:6379/"
+                      :prod #env REDIS_URL}
+ :host #profile {:dev "localhost"
+                 :prod "versions.deps.co"}
+ :port #profile {:dev 8080
+                 :prod 80}}
```
Comment (dotemacs) [inline resources/config.edn]: I'm guessing that `:prod` should be set to 443 instead of 80, seeing as it's being used on https

Comment (danielcompton) [reply resources/config.edn]: 80 is fine, because it's running on Heroku who handle the SSL.

**Comment by dotemacs** [comment]
Thanks for the confirmation. 

Is there anything else?

Thanks

**Comment by danielcompton** [comment]
Nope, that's all great, I've merged this now. Thanks heaps, I really appreciate this!

**Comment by dotemacs** [comment]
No problem, thank you for running this service

**Pull request merged**